openhands/runtime/impl/remote/remote_runtime.py

```diff
@@ -379,7 +396,7 @@ def run_action(self, action: Action) -> Observation:
                 )
             return obs
 
-    def _send_request(self, method, url, **kwargs):
+    def _send_request(self, method, url, is_retry=False, **kwargs):
```

Comment:
Just a small odd thing, this is by default `False`, and all I see is `False` being sent, so when is it supposed to be true? Maybe the default should be?

Reply:
it should only be true if it's being called from inside this function

I had to set it False explicitly in most other cases because of the `kwargs` plumbing, sadly

Reply:
Ah I see! That's where it was hiding. 🙊